auth_signup_verify_email/controllers/main.py

```diff
@@ -34,7 +34,7 @@ def passwordless_signup(self, values):
             values["email"] = values.get("login")
 
         # Remove password
-        values["password"] = False
+        values["password"] = ""
```

Comment:
OH I see, thanks! 😊 It's because of the secret computation. It needs strings to work.